x-pack/plugins/security_solution/scripts/run_cypress/utils.ts

```diff
@@ -52,28 +48,56 @@ export const parseTestFileConfig = (
       return {};
     }
 
-    return _.reduce(
-      ftrConfig.value.properties,
-      (acc: Record<string, string | number | Record<string, string>>, property) => {
-        const key = (property.key as Identifier).name;
-        let value;
-        if (property.value.type === 'ArrayExpression') {
-          value = _.map(property.value.elements, (element) => {
-            if (element.type === 'StringLiteral') {
-              return element.value as string;
-            }
-            return element.value as string;
-          });
-        } else if (property.value.type === 'StringLiteral') {
-          value = property.value.value;
-        }
-        if (key && value) {
-          acc[key] = value;
-        }
-        return acc;
-      },
-      {}
-    );
+    const ftrConfigCode = generate(ftrConfig.value, { jsonCompatibleStrings: true }).code;
+
+    try {
+      // TODO:PT need to assess implication of using this approach to get the JSON back out
+      const ftrConfigJson = new Function(`return ${ftrConfigCode}`)();
```

Comment:
I know. Scary `eval()` code here.  Because this is for testing and because usernames / passwords are already all in the clear, I think this is ok... but would like some feedback.